src/StoreApi/Authentication.php

```diff
@@ -1,6 +1,8 @@
 <?php
```

Comment:
Let's add a doc page to `src/StoreApi/docs` explaining how this works, the filters in action, and the logic we have.

Reply:
[Doc](https://github.com/woocommerce/woocommerce-blocks/pull/5962/commits/a8bd4fc22e3884b325571739829a116933b249ce)